projects/1.20/assets/bovines-and-buttercups/bovinesandbuttercups/lang/zh_cn.json

```
@@ -0,0 +1,62 @@
+{
+    "advancements.husbandry.bovinesandbuttercups.prevent_an_effect.title": "状态免除",
+    "advancements.husbandry.bovinesandbuttercups.prevent_an_effect.description": "使用花蜜避免状态效果的影响。",
+    "advancements.husbandry.bovinesandbuttercups.lock_an_effect.title": "状态锁定",
+    "advancements.husbandry.bovinesandbuttercups.lock_an_effect.description": "使用花蜜锁定状态效果的持续时间。",
+    "advancements.husbandry.bovinesandbuttercups.mutate_a_moobloom.title": "斑斓银河",
+    "advancements.husbandry.bovinesandbuttercups.mutate_a_moobloom.description": "繁殖出变异的哞花子代。",
+    
+    "block.bovinesandbuttercups.buttercup": "毛茛",
+    "block.bovinesandbuttercups.pink_daisy": "粉色雏菊",
+    "block.bovinesandbuttercups.limelight": "水亚木",
+    "block.bovinesandbuttercups.bird_of_paradise": "天堂鸟",
+    "block.bovinesandbuttercups.chargelily": "青百合",
+    "block.bovinesandbuttercups.hyacinth": "风信子",
+    "block.bovinesandbuttercups.snowdrop": "雪花莲",
+    "block.bovinesandbuttercups.tropical_blue": "热带蓝",
+    "block.bovinesandbuttercups.freesia": "小苍兰",
+    "block.bovinesandbuttercups.potted_buttercup": "毛茛盆栽",
+    "block.bovinesandbuttercups.potted_pink_daisy": "粉色雏菊盆栽",
+    "block.bovinesandbuttercups.potted_limelight": "水亚木盆栽",
+    "block.bovinesandbuttercups.potted_bird_of_paradise": "天堂鸟盆栽",
+    "block.bovinesandbuttercups.potted_chargelily": "青百合盆栽",
+    "block.bovinesandbuttercups.potted_hyacinth": "风信子盆栽",
+    "block.bovinesandbuttercups.potted_snowdrop": "雪花莲盆栽",
+    "block.bovinesandbuttercups.potted_tropical_blue": "热带蓝盆栽",
+    "block.bovinesandbuttercups.potted_freesia": "小苍兰盆栽",
+    
+    "block.bovinesandbuttercups.custom_flower": "未知的花",
+    "block.bovinesandbuttercups.missing_flower": "未知的花",
+    "block.bovinesandbuttercups.custom_mushroom": "未知的蘑菇",
+    "block.bovinesandbuttercups.missing_mushroom": "未知的蘑菇",
+    "block.bovinesandbuttercups.custom_mushroom_block": "未知的蘑菇方块",
+    "block.bovinesandbuttercups.missing_mushroom_block": "未知的蘑菇方块",
+    "block.bovinesandbuttercups.custom_potted_flower": "未知的花盆",
+    "block.bovinesandbuttercups.custom_potted_mushroom": "未知的花盆",
+    
+    "item.bovinesandbuttercups.nectar_bowl": "花蜜碗",
+    "item.bovinesandbuttercups.moobloom_spawn_egg": "哞花刷怪蛋",
+    
+    "entity.bovinesandbuttercups.moobloom": "哞花",
+    "subtitles.entity.moobloom.convert": "哞花转变",
+    "subtitles.entity.moobloom.eat": "哞花进食",
+    "subtitles.entity.moobloom.milk": "挤哞花奶",

```

```suggestion
    "subtitles.entity.moobloom.convert": "哞花：转变",
    "subtitles.entity.moobloom.eat": "哞花：进食",
    "subtitles.entity.moobloom.milk": "哞花：被挤奶",
```
见https://zh.minecraft.wiki/w/%E5%AD%97%E5%B9%95 中哞菇部分